front_end/src/components/searchResultOverview/searchResultOverview.js

```diff
@@ -10,13 +10,49 @@ import {
   TablePagination
 } from "@material-ui/core";
 import SearchWidget from "../landing_page/search_widget/SearchWidget";
+import { connect } from "react-redux";
+import { displayResultsOverview } from "../../actions/searchResultActions";
 
 class searchResultOverview extends Component {
   render() {
+    const result = this.props.result.result;
+    let hotels;
+
     const styles = {
       rating: { float: "left", width: "100%" },
       imageStyle: { margin: 20, width: 200, height: 200, float: "left" }
     };
+    if (result.length) {
+      hotels = result.map(hotel => {
+        return (
+          <Card style={{ marginBottom: 10 }}>
+            <CardMedia
+              style={styles.imageStyle}
```

Comment:
4
Here, instead of using styles={styles.imageStyle}, you use className={classes.imageStyle}
You will have to do this in all of your tags/components that use your custom css.

Reply:
replaced

Reply:
how about the style in <Card style={{ marginBottom: 10 }}>? Is there a difference?

Reply:
I think the whole CSS specificity thing still applies, but since it's a small margin change it won't cause problems.